packages/feedback-service/src/typedef.graphql

```diff
@@ -44,7 +76,7 @@ type TicketType {
 
 type Query {
   # Fetches all Feedbacks
-  listFeedback: [FeedbackType]
+  listFeedback: [FeedbackWithUserType]
   # Fetches specific Feedback by its id
   getFeedback(id: String!): FeedbackType
```

Comment:
You also need to do the same for getFeedback and any other queries that are returning feedback.